Description: Use /usr/share/wit instead of /usr/local/share/wit
 This is hardcoded in the code.
Author: Michael Stapelberg <stapelberg@debian.org>
Last-Update: 2013-07-11

---

Index: wit/src/lib-std.c
===================================================================
--- wit.orig/project/src/lib-std.c
+++ wit/project/src/lib-std.c
@@ -547,7 +547,7 @@ void SetupLib ( int argc, char ** argv,
     TRACE("PROC-PATH: %s\n",proc_path);
 
     static char share[] = "/share/wit/";
-    static char local_share[] = "/usr/local/share/wit/";
+    static char local_share[] = "/usr/share/wit/";
 
     char path[PATH_MAX];
     if (readlink(proc_path,path,sizeof(path)))
